src/inpage/rpcCall.ts

```diff
@@ -0,0 +1,39 @@
+import { API_TYPE } from '../constants';
+import { requestExtensionAPI } from './utils';
+import { IRPCCallResponsePayload, IRPCCallRequestPayload } from '../types';
+
+const processingRPCCallRequests: { [id: string]: IRPCCallRequest } = {};
+
+export function rpcCall(method: string, args: any[]): Promise<string> {
```

Comment:
should be to do this with `async/await` instead of having to wrap it in a promise. its newer syntax and easier to read.